### DISCUSSION TITLE
May I know how NFDI4PSO was chosen for the prefix?

### DISCUSSION THREAD

**Discussion by xiaoranzhou** [nfdi4plants/nfdi4plants_ontology#17] [Q&A]
Hi, I will be very happy if the following questions can be answered:
1. How is “NFDI4PSO” being chosen as the prefix of terms and the acronym of the DataPLANT Ontology? 
2. Was there a discussion or decision about it?
3. The EDAM uses a url as the term ID, was it also considered in the “NFDI4PSO”?

- For the first two questions: . ”NFDI4PSO” is comparatively long as a prefix. Plant Ontology has “PO” as prefix (PO:0030059), Plant Trait Ontology has “TO” as prefix. Other ontologies mostly have a prefix less than four letters, “NFDI4PSO” has eight letters. 

- For the last question: The advantage of using a url as ID is that the users can know more information regardless of the tools they use. The disadvantage is that the urls have more letters so they are difficult to remember for some people.

**Comment by muehlhaus** [comment]
This are excellent points, that need a dedicated discussion with heavy involvement and also guidance of TA1.
